frontend/src/scenes/session-recordings/SessionRecordingsTable.tsx

```diff
@@ -175,20 +177,23 @@ export function SessionRecordingsTable({ personUUID, isPersonPage = false }: Ses
                         </div>
                     )}
                 </div>
-                <Button
-                    style={{ display: showFilters ? 'none' : undefined }}
-                    onClick={() => {
-                        enableFilter()
-                        if (isPersonPage) {
-                            const entityFilterButtons = document.querySelectorAll('.entity-filter-row button')
-                            if (entityFilterButtons.length > 0) {
-                                ;(entityFilterButtons[0] as HTMLElement).click()
+                {!showFilters && (
+                    <LemonButton
+                        type="stealth"
```

Comment:
I think this "Filter recordings" one should be `secondary` so as not to be _too_ stealthy, WDYT?
Also, it's noticeably taller than the Ant buttons on the right, but this can be fixed by just making this one `compact`.
<img width="995" alt="Screen Shot 2022-03-14 at 13 30 15" src="https://user-images.githubusercontent.com/4550621/158172381-996f70dc-1eb5-4e3e-987e-1f2b8d157a81.png">
```suggestion
                        type="secondary"
                        compact
```